### DISCUSSION TITLE
after orientation program my details got erased ,now again doing it but it not submitting ,and i'm unable to open the self learning program also, what i have to do?

### DISCUSSION THREAD

**Discussion by shahshaik2000** [github/india#264] [General]
after orientation program my details got erased ,now again doing it but it not submitting ,and i'm unable to open the self learning program also, what i have to do?

**Comment by tripti6ty** [comment]
you can open FRT portal in incognito mode or open the portal after clearing browsing history and cache in normal  mode.
you'll find details are already saved.

**Comment by tripti6ty** [comment]
you can open FRT portal in incognito mode or open the portal after clearing browsing history and cache in normal  mode.
you'll find details are already saved.

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
